src/containers/ForgotPassword/index.js

```diff
@@ -1,8 +1,41 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import React, { Component } from 'react';
+import ForgotPassword from './forgotPass';
+import schema from './utils/schema';
 
-const ForgotPassword = props => {
-  return <UnderConstruction {...props} />;
-};
+export default class index extends Component {
+  // eslint-disable-next-line react/state-in-constructor
+  state = {
+    email: '',
+    error: '',
+  };
 
-export default ForgotPassword;
+  handleChange = ({ target: { value } }) => {
+    const { email } = this.state;
+    this.setState({ email: value }, async () => {
+      const valid = await schema.isValid({ email: email.trim() });
+      if (!valid) {
+        this.setState({ error: 'the input is not a valid email' });
+      } else {
+        this.setState({ error: '' });
+      }
+    });
+  };
+
+  handleClick = () => {
+    // Logic will be added hereW
```

Comment:
```suggestion
    // Logic will be added here
```
it's will be good if you write a cleaner comment like: make a  fetch to firebase server :smiley: